libs/game/tilemap.ts

```diff
@@ -105,7 +100,16 @@ namespace tiles {
         }
 
         setMap(map: Image) {
-            this._map = map;
+            if (!this._map) {
```

Comment:
what if map is undefined?

Reply:
It gets handled below in line 110, but I should just check that first and return early in that case; no reason to set up if there is no map